zcash_client_sqlite/src/wallet/init/migrations/add_account_uuids.rs

```diff
@@ -77,8 +79,6 @@ impl RusqliteMigration for Migration {
                   OR
                   (
                     account_kind = {account_kind_imported}
-                    AND hd_seed_fingerprint IS NULL
-                    AND hd_account_index IS NULL
```

Comment:
Document that this migration is now not just creating account UUIDs, but also allowing the imported account kind to have derivation metadata.